vagrant/Vagrantfile

```diff
@@ -44,13 +45,16 @@ Vagrant.configure(2) do |config|
     aws.session_token = ENV['AWS_SESSION_TOKEN']
     aws.associate_public_ip = true
     aws.tags = { 'Name' => (ENV['DEPLOY_ENV'] || ENV['USER']) + " concourse" }
+    aws.keypair_name = ENV['VAGRANT_SSH_KEY_NAME']
```

Comment:
👌  could you add something to the commit message which explains that we need to generate/delete an SSH keypair because the Ubuntu AMI now that we're now using no longer contains the default Vagrant insecure key?